udapi/core/bundle.py

```diff
@@ -7,7 +7,12 @@
 from root import Root
 
 class Bundle(object):
-    """Bundle can be used for embracing two or more Universal Dependency trees that are associated in some way (e.g. parallel translations) inside a document. Unless different zones are differentiated in a bundle, there's only one tree per bundle by default."""
```

Comment:
But this line comes from @zabokrtsky and my commit replaces this with several shorter lines :-)